iroh-net/src/hp/netcheck/reportgen/probes.rs

```diff
@@ -724,4 +674,213 @@ mod tests {
+                    probes: vec![
+                        Probe::Icmp {
+                            delay: Duration::from_micros(414_400),
+                            node: derp_node_2.clone(),
+                        },
+                        Probe::Icmp {
+                            delay: Duration::from_micros(466_800),
+                            node: derp_node_2.clone(),
+                        },
+                    ],
+                },
+            ]
+            .into_iter()
+            .collect();
+            if if_state.have_v6 {
+                expected_plan.add(ProbeSet {
+                    name: "region-1-stunipv6".into(),
+                    proto: ProbeProto::StunIpv6,
+                    probes: vec![
+                        Probe::StunIpv6 {
+                            delay: Duration::ZERO,
+                            node: derp_node_1.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(52_400),
+                            node: derp_node_1.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(104_800),
+                            node: derp_node_1.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(157_200),
+                            node: derp_node_1.clone(),
+                        },
+                    ],
+                });
+                expected_plan.add(ProbeSet {
+                    name: "region-2-stunipv6".into(),
+                    proto: ProbeProto::StunIpv6,
+                    probes: vec![
+                        Probe::StunIpv6 {
+                            delay: Duration::ZERO,
+                            node: derp_node_2.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(52_400),
+                            node: derp_node_2.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(104_800),
+                            node: derp_node_2.clone(),
+                        },
+                        Probe::StunIpv6 {
+                            delay: Duration::from_micros(157_200),
+                            node: derp_node_2.clone(),
+                        },
+                    ],
+                })
+            }
+
+            println!("{} round", i);
+            println!("expected:");
+            println!("{expected_plan}");
+            println!("actual:");
+            println!("{plan}");
+            // Just in case there's a bug in the Display impl:
+            assert_eq!(plan, expected_plan, "{}", i);
+
+            // The readable error:
+            assert_eq!(plan.to_string(), expected_plan.to_string(), "{}", i);
```

Comment:
these two asserts seem to have flipped around?  also not sure if the diff is now just entirely hidden by the round number or not.